src/openvisuspy/slice.py

```diff
@@ -1219,8 +1302,8 @@ def gotNewData(self, result):
 
 			# in dynamic mode, I need to use the data range
 			if mode=="dynamic":
-				self.range_min.value = data_range[0]
-				self.range_max.value = data_range[1]
+				self.range_min.value = round(data_range[0],6) # I am trying to avoid too many refreshes

```

Comment:
It seems reasonable, but what for datasets which are quickly moving in some spot in a range <1e-6? Not sure if this is too much dataset-dependent.